tests/basic_functionality.rs

```diff
@@ -188,3 +188,72 @@ fn basic_read_write_round_trip() {
         .expect("writing should succeed");
     assert_eq!(buf1, buf2);
 }
+
+#[test]
+fn report_with_j1_0_and_j2_1_generates_one_tcn() {
+    let rak = ReportAuthorizationKey::new(rand::thread_rng());
+
+    let signed_report = rak
+        .create_report(MemoType::CoEpiV1, b"symptom data".to_vec(), 0, 1)
+        .expect("Report creation can only fail if the memo data is too long");
```

Comment:
We can cover "memo too long" with a test. 
Also to high values for j1, j2.

Reply:
I added `report_with_j2_max_and_j1_max_minus_1_generates_2_tcns` to test max values (`u16::MAX - 1`, `u16::MAX`). Maybe it makes sense to add (`u16::MAX`, `u16::MAX`) too.